### PR TITLE
Swathi

### PR DESCRIPTION
**Pull request by Swathibhat01**
@chethan

### CODE REVIEW

**Comment by SwathiAR** [comment]
@chethan

File: src/main/java/Basic99/AlphanumericCounter.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package Basic99;
+
+/**
+ * Created by root on 12/8/15.
+ */
+public class AlphanumericCounter {
+
+    public int countAlphanumericValues(String str){
+        if(str==null || str.equals(" ")){
```
Comment (chethan) [inline src/main/java/Basic99/AlphanumericCounter.java:9]: Replace this and similar code in all the classes with a util method.

File: src/main/java/Basic99/AlphanumericCounter.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            return  0;
+        }
+        int count = 0;
+
+        for(int i = 0 ; i<str.length();i++){
+            if((str.charAt(i) > 64 && str.charAt(i) < 91) ||
```
Comment (chethan) [inline src/main/java/Basic99/AlphanumericCounter.java:15]: Avoid magic numbers

File: src/main/java/Basic99/ChessBoard.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+/**
+ * Created by root on 12/10/15.
+ */
+public class ChessBoard {
+    public String[][] createChessBoard() {
+        String[][] chessBoard = new String[8][8];
+
+        for (int i = 0; i < 8; i++) {
+            if (i % 2 == 0) {
+                for (int j = 0; j < 7; j += 2) {
```
Comment (chethan) [inline src/main/java/Basic99/ChessBoard.java:12]: Pull the condition statement in and loop outside

File: src/main/java/Basic99/Coprimes.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+/**
+ * Created by root on 12/9/15.
+ */
+public class Coprimes {
+
+    public boolean isCoprime(int num1 , int num2){
+        Euclid e = new Euclid();
+        int gcd = e.getGCD(num1, num2);
+        if(gcd==1){
```
Comment (chethan) [inline src/main/java/Basic99/Coprimes.java:11]: Compress condition to return statement

File: src/main/java/Basic99/DigitToWords.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * Created by root on 12/10/15.
+ */
+public class DigitToWords {
+    public String[] getWords(String digits){
+        String[] words = new String[digits.length()];
+        String[] givenWords = {"Oh" , "One" , "Two" , "Three" , "Four" , "Five" , "Six" , "Seven" ,"Eight" , "Nine"};
+
+        for(int i = 0 ; i<digits.length();i++){
+            int c = (int)digits.charAt(i)-48;
```
Comment (chethan) [inline src/main/java/Basic99/DigitToWords.java:12]: Magic numbers

File: src/main/java/Basic99/Euclid.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+/**
+ * Created by root on 12/9/15.
+ */
+public class Euclid {
+    public int getGCD(int num1 , int num2){
+
+        if(num1==0 ){
+            return num2;
+        }
```
Comment (chethan) [inline src/main/java/Basic99/Euclid.java:11]: Remove unnecessary else conditions

File: src/main/java/Basic99/GoldBachPair.java [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+/**
+ * Created by root on 12/10/15.
+ Goldbach’s conjecture says that every positive even number greater than 2 is the sum of two prime numbers. Example: 28 = 5 + 23. It is one of the most famous facts in number theory that has not been proved to be correct in the general case. It has been numerically confirmed up to very large numbers. Write a predicate to find the two prime numbers that sum up to a given even number.
+
+
+ */
+public class GoldBachPair {
+    Prime p = new Prime();
+
+public int[] getGoldBachPair(int num){
```
Comment (chethan) [inline src/main/java/Basic99/GoldBachPair.java:12]: Replace getGoldBachPair(int num) with getGoldBachPair(int num,2)

File: src/main/java/Basic99/LetterECounter.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+    public int countE(String str){
+        if(str==null || str.equals(" ")){
+            return 0;
+        }
+        int count=0;
+
+
+        for(int i = 0 ; i<str.length() ; i++){
+            if(str.charAt(i)=='e'){
```
Comment (chethan) [inline src/main/java/Basic99/LetterECounter.java:16]: count  upper case 'E' also

**Pull request merged**